examples/python_scripts/stacked_line_graphs.py

```diff
@@ -0,0 +1,85 @@
+import sys
+
+sys.path.append("/usr/gapps/spot/dev/hatchet-venv/x86_64/lib/python3.9/site-packages/")
+sys.path.append("/g/g20/hao3/thicket")
+sys.path.append("/usr/gapps/spot/dev/hatchet/x86_64/")
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import hatchet as ht
+import thicket as th
+import copy
+import re
+import argparse
+
+from IPython.display import display, HTML
+from glob import glob
+from hatchet import QueryMatcher
+
+
+def arg_parse():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--input_files", default="", type=str)
```

Comment:
Arguments that are required, such as `input_files`, should have no `default` and `required=True` with a `help` description. It would also be nice to have a program description like this example so users can run `python stacked_line_graphs.py --help`
```suggestion
    parser = argparse.ArgumentParser(
        prog="stacked_line_graphs.py",
        description="Generate stacked line graphs from Caliper files.",
        epilog="This script reads in Caliper files and generates stacked line graphs based on the specified parameters.",
    )
    parser.add_argument("--input_files", required=True, type=str, help="Directory of Caliper files to read in.")
```